packages/gatsby/src/cache/wrap-callback.ts

```diff
@@ -37,8 +37,8 @@ module.exports = function wrapCallback<T>(
       cb = args.pop()
     }
 
-    // eslint-disable-next-line @babel/no-invalid-this
-    const promise = fn.apply(this, args)
+    // @ts-ignore - unsure if fixing this introduces problems
+    const promise = fn.apply(this, args) // eslint-disable-line @babel/no-invalid-this
```

Comment:
wouldn't eslint-disable-next-line still work because next line is also a comment?

Reply:
No, it didn't work for me so I had to move it :/